packages/default/scss/editor/_layout.scss

```diff
@@ -66,6 +66,12 @@
         flex-wrap: wrap;
         position: relative;
 
+        &.k-toolbar-resizable {
+            flex-wrap: nowrap;
+            overflow: hidden;
+            flex: 1;
```

Comment:
The [`flex` shorthand defaults have changed](https://github.com/philipwalton/flexbugs#flexbug-6) between IE10 and 11, use the full `flex: 1 1 0%` instead.